composer.json

```diff
@@ -12,7 +12,8 @@
     "require": {
         "php": "^8.2",
         "ext-pdo": "*",
-        "canyon-gbs/case-module": "*",
+        "canyon-gbs/assist-data-model": "^1.0",
+        "canyon-gbs/case": "^1.0",
```

Comment:
Should these be versioned at this point?

Reply:
Ah no, that was a mistake with something I was doing.